manifests/l/LeNgocKhoa/Laragon/5.0.0.210523/LeNgocKhoa.Laragon.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: LeNgocKhoa.Laragon
+PackageVersion: 5.0.0.210523
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://github.com/leokhoa/laragon/releases/download/5.0.0/laragon-wamp.exe
+  InstallerSha256: C3D72330A2FBE27614A7C71355D297C88FD192A2B7C96133CF535D3AB95C27F1
+  InstallerSwitches:
+    Custom: /NORESTART
+  UpgradeBehavior: install
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://github.com/leokhoa/laragon/releases/download/5.0.0/laragon-wamp.x86.exe
+  InstallerSha256: B984A00619C321AA95492F840C183A6C58CFCC3522F11B795E302941D3EE5211
+  InstallerSwitches:
+    Custom: /NORESTART
+  UpgradeBehavior: install
```

Comment:
```
  Architecture: x86
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://github.com/leokhoa/laragon/releases/download/5.0.0/laragon-wamp.x86.exe
  InstallerSha256: B984A00619C321AA95492F840C183A6C58CFCC3522F11B795E302941D3EE5211
  InstallerSwitches:
    Custom: /NORESTART
  UpgradeBehavior: install
```
I don't have a 32-bit VM for Windows to test this but have you tried installing this on a 32-bit OS and see what happens or shows up?

Installing 32-bit applications on a 32-bit OS provides different results compared to installing 32-bit applications on a 64-bit OS.

Reply:
![Screenshot 2021-07-24 145748](https://user-images.githubusercontent.com/69106310/126869177-08f27269-a206-4a3b-a4aa-b6f1bda3a2a6.png)